packages/nyc-test-coverage/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@temporalio/nyc-test-coverage",
```

Comment:
[istanbul](https://npmjs.com/package/istanbul) is technically deprecated, and nyc is the replacement. However, under the hood, nyc uses the same core code coverage logic as istanbul (like istanbul-lib-coverage).

Reply:
So why are we using istanbul if it's deprecated?
I'm not following.. either call this `istanbul-test-coverage` and use istanbul or call it `nyc-test-coverage` and use `nyc`.
Am I missing something?

Reply:
Istanbul isn't deprecated. The istanbul CLI is.

![image](https://user-images.githubusercontent.com/251288/182980094-053c02e4-2d7c-4df4-a81d-1890f96d482f.png)

![image](https://user-images.githubusercontent.com/251288/182980098-24aedd0f-d203-4f92-bd90-859c07cdcd00.png)

Reply:
@lorensr is right.

"I'm not following.. either call this istanbul-test-coverage and use istanbul or call it nyc-test-coverage and use nyc." <-- we're calling this package nyc-test-coverage because it's meant to be used with nyc.

Reply:
Got it, would be nice to mention this in the package README (which is missing BTW).